schema/src/main/scala/io/shiftleft/codepropertygraph/schema/CallGraph.scala

```diff
@@ -35,13 +37,13 @@ object CallGraph extends SchemaBase {
                     |ignored. It is suggested to set it to -1.
                     |""".stripMargin
       )
+      .mandatory(PropertyDefaults.Int)
```

Comment:
I think -2 would be more sensible -- but the docstring suggests -1 when this is not applicable; both need to be consistent.

Reply:
i changed it to explicit `-1` as per the docstring - @fabsx00 @ml86 wdyt?